src/utils/temporary/generateEAT.ts

```diff
@@ -60,7 +60,7 @@ export const getEATForMulticall = async ({
 export const baseUrlByEnvironment = (environment: string) => {
   switch (environment) {
     case 'local':
-      return 'http://localhost:4000'
+      return 'http://localhost:8080'
```

Comment:
Intended?

Reply:
Yes, it seemed like the endpoint for local changed for violet app so I couldn't get it working without changing it here. I don't remember changing any config on my violet-app side so I thought it was a change that didn't propagate to mauve-dex.

Do you think it's just a misconfiguration on my setup, and it should be hosting at 4000 locally?

Reply:
I remember someone unintentionally changed to 4000, but then on the launch week @ttollers did other things to adjust it for being 4000 as default (which I don't remember what exactly)

summary is: I think we should leave it as it is otherwise we would need to search and do those other changes back 😂

Reply:
Ok 😂 will not touch this

Reply:
Fixed in https://github.com/violetprotocol/mauve-dex/pull/153/commits/43baa8bf3a8eb2c0c82bfcbe8458a4b97dd0908a

Reply:
The only thing I did was add a client id that works locally with 4000 😄